metadata-events/mxe-avro-1.7/build.gradle

```diff
@@ -7,7 +7,9 @@ apply plugin: 'java'
 
 dependencies {
   compile externalDependency.avro_1_7
-  compile externalDependency.avroCompiler_1_7
+  compile(externalDependency.avroCompiler_1_7) {
+    exclude group: 'org.apache.velocity', module: 'velocity'
```

Comment:
do we know of any adverse effects of excluding this altogether? causes potential for no class found exceptions at runtime

Reply:
(but maybe avroCompile only used at build time?)

Reply:
So at runtime we're including the updated version, in the WAR runtimeClasspath Velocity-Core 2.3 is present (per dependencyInsight). This excludes the legacy Velocity dep and would impact build time only (barring 2.3 actually being incompatible in which case we cannot upgrade it in place and would need to update to Avro 1.9+)